core/server/common/src/main/java/alluxio/ProcessUtils.java

```diff
@@ -60,7 +60,7 @@ public static void run(Process process) {
    * @param args args for the format string
    */
   public static void fatalError(Logger logger, String format, Object... args) {
-    fatalError(logger, null, format, args);
+    fatalError(logger, new Throwable(), format, args);
```

Comment:
why is the new Throwable needed here? seems like the called method handles null situation?